R/parents_of_children.R

```diff
@@ -0,0 +1,74 @@
+require(data.table)
+
+#' @name parents_of_children
+#' 
+#' @description Vectorized implementation of "parents_of_child"
+#' 
+#' @param child_location_id [int] Location ID to pull parent ID for
+#' @param hierarchy [data.table] Hierarchy. Must have columns location_id, path_to_top_parent, and level.
+#' @param parent_level [int] Level of the parent hierarchy to query.
+#' @export 
+parents_of_children <- function(  
+  child_loc_ids, 
+  hierarchy, 
+  parent_level
+){
+  return(sapply(child_loc_ids, function(x) parents_of_child(x, hierarchy, parent_level)))
+}
+
+#' @name parents_of_child
+#' 
+#' @description Given a location ID, returns its parent ID at a given level. 
+#' 
+#' @param child_location_id [int] Location ID to pull parent ID for
+#' @param hierarchy [data.table] Hierarchy. Must have columns location_id, path_to_top_parent, and level.
+#' @param parent_level [int] Level of the parent hierarchy to query.
+#' 
+#' @export
+#' @return 
+parents_of_child <- function(
```

Comment:
prefer `parent_of_child` for clarity - each child should only return single parent at a single level

Reply:
Done.